src/frontend/apps/impress/src/features/left-panel/components/LeftPanel.tsx

```diff
@@ -42,27 +77,42 @@ export const LeftPanel = ({ children }: PropsWithChildren) => {
             border-right: 1px solid ${colors['greyscale-200']};
         `}
         >
-          <div>
-            <SeparatedSection>
-              <Box
-                $padding={{ horizontal: 'sm' }}
-                $direction="row"
-                $justify="space-between"
-                $align="center"
-              >
-                <Box $direction="row" $gap="2px">
-                  <Button
-                    onClick={goToHome}
-                    size="medium"
-                    color="primary-text"
-                    icon={<Icon iconName="house" />}
-                  />
-                </Box>
-                <Button onClick={createNewDoc}>{t('New doc')}</Button>
+          {getContent()}
+        </Box>
+      )}
+
+      {!isDesktop && (
+        <Box
+          $css={css`
+            z-index: 999;
+            width: 100dvw;
+            height: calc(100dvh - 52px);
+            border-right: 1px solid var(--c--theme--colors--greyscale-200);
+            position: fixed;
+            left: ${mobileMenuIsOpen ? '0' : '-100vw'};
```

Comment:
You get better perf with:
```suggestion
            transform: translateX(${mobileMenuIsOpen ? '0' : '-100%'});
```